react-pr2/public/locales/en/translation.json

```
@@ -3,5 +3,11 @@
   "Login": "Login",
   "Logout": "Logout",
   "Search": "Search...",
-  "Account": "Account"
+  "Account": "Account",
+  "PRODUCT CATEGORY": "PRODUCT CATEGORY",
```

```suggestion
  "Product category": "PRODUCT CATEGORY",
```
Để key bình thường là được rồi nhé